### PR TITLE
Add memcache include file

### PR DESCRIPTION
**Pull request by ben-hamelin**
## Summary

Adds the ACSF required memcache include.

## Metadata

| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes
| Did you apply meaningful labels to the pull request? | yes
| Did you perform a self review first? | yes
| Documentation reflects changes? | no
| `CHANGELOG` reflects changes? | yes
| Unit/Functional tests reflect changes? | no
| Did you perform browser testing? | yes
| Risk level | Medium
| Relevant links | https://thinkoomph.jira.com/browse/RIG-171

### CODE REVIEW

File: factory-hooks/post-settings-php/acsfd8+.memcache.settings.php [hunk 1]
```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * @file
+ * Contains caching configuration.
+ */
+use Composer\Autoload\ClassLoader;
```
Comment (pfrilling) [inline factory-hooks/post-settings-php/acsfd8+.memcache.settings.php:6]: Do we want to add strict type checking to this file? I believe it came from Acquia.

File: factory-hooks/post-settings-php/acsfd8+.memcache.settings.php [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+ */
+
+if (getenv('AH_SITE_ENVIRONMENT') &&
+  array_key_exists('memcache', $settings) &&
+  array_key_exists('servers', $settings['memcache']) &&
+  !empty($settings['memcache']['servers'])
+) {
+
+// Check for PHP Memcached libraries.
+$memcache_exists = class_exists('Memcache', FALSE);
```
Comment (pfrilling) [inline factory-hooks/post-settings-php/acsfd8+.memcache.settings.php:25]: Should this code block be indented since it's wrapped in an if?

**Review by pfrilling — CHANGES_REQUESTED** [review]
Minor coding standard issues.

File: factory-hooks/post-settings-php/acsfd8+.memcache.settings.php (hunk 1 above)
Comment (ben-hamelin) [reply factory-hooks/post-settings-php/acsfd8+.memcache.settings.php:10]: Yep!

File: factory-hooks/post-settings-php/acsfd8+.memcache.settings.php (hunk 2 above)
Comment (ben-hamelin) [reply factory-hooks/post-settings-php/acsfd8+.memcache.settings.php]: Got it

**Comment by ben-hamelin** [comment]
Ok addressed feedback

**Review by pfrilling — APPROVED** [review]
Looks good.

**Pull request merged**